openassessment/xblock/staff_area_mixin.py

```diff
@@ -319,11 +319,14 @@ def get_student_submission_context(self, student_username, submission):
         Returns:
             A context dict for rendering a student submission and associated rubric (for staff grading).
         """
+        user_preferences = get_user_preferences(self.runtime.service(self, 'user'))
+
         context = {
             'submission': create_submission_dict(submission, self.prompts) if submission else None,
             'rubric_criteria': copy.deepcopy(self.rubric_criteria_with_labels),
             'student_username': student_username,
-            'time_zone': get_current_time_zone(self.runtime.service(self, 'user')),  # localize for staff user
+            'user_timezone': user_preferences['user_timezone'],  # localize for staff user
```

Comment:
The `localize for staff user` comment makes more sense to stay with the call to `get_user_preferences` (now moved above).  Since the comment refers to _which_ user's preferences are used.